packages/generators/admin/component/templates/tests/index.test.js.hbs

```diff
@@ -0,0 +1,63 @@
+/**
+ *
+ * Tests for {{name}}
+ *
+ */
+
+import React from 'react';
+import { render {{~#if useRedux}} as tlRender {{~/if}} } from '@testing-library/react';
+{{#if useRedux}}
```

Comment:
I thought about it but I would prefer to add a custom action type to fix linter errors automatically, what do you think?
It will be easier to maintain templates if we do not have to think about the current linter config.

Reply:
it's up to you but I think it would take less time to write them down.

Reply:
Do you plan on doing it in another PR?

Reply:
It has been added in this PR, very little changes actually.